specifications/passive/android_phone-1.0.0.yml

```diff
@@ -156,3 +156,11 @@ data:
     value_schema: .passive.phone.PhoneUserInteraction
     sample_rate:
       dynamic: true
+  # Phone Audio Input
+  - type: PHONE_AUDIO_INPUT
```

Comment:
I think this should be in a separate specifications file

Reply:
Yeah i am not sure (they are likely closely related to the phone). For audio input, it is pretty separate (similar to opensmile audio).